src/System.Collections.Concurrent/tests/ConcurrentDictionary/ConcurrentDictionaryTests.cs

```diff
@@ -546,25 +546,51 @@ public static void TestDebuggerAttributes()
         }
 
         [Fact]
-        public static void TestConstructor_Negative()
+        [SkipOnTargetFramework(TargetFrameworkMonikers.NetFramework, ".NET Framework hasn't received the fix for https://github.com/dotnet/corefx/issues/18432 yet.")]
+        public static void TestNullComparer()
```

Comment:
@stephentoub That sounds good. I'm not thinking of a good way to differentiate in a test whether it was using the default comparer or if it was falling back to using Object.GetHashCode/Equals directly. How could I do that?